src/components/SubHeader/subHeader.css

```diff
@@ -24,38 +29,40 @@
     color: #FFFFFF;
   }
   
-  .headerNormal {
+  .subHeader_content {
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding: 25px 0px;
+    padding: 25px;
+  }
+
+  .subHeader_content .content {
+    text-align: center;
   }
   
-  .headerText {
+  .subHeader_content .content p {
```

Comment:
`subHeader_content .content p { ...` :no_entry: 
Please try to avoid directly applying styles to HTML elements.
always give class_names to elements that will require styling.